result/isislsp.yaml

```diff
@@ -0,0 +1,352 @@
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.IsisRouter/properties/name"
+        lsp_filters:
+          description: >-
+            Specify which LSPs to return. If the list is empty or missing   
+            then all LSPs will be returned, e.g. '6400000000010000'.
+            LSP ID is consist of the System ID of a neighbor, the Pseudonode ID, and the LSP number of the LSPs.
+            The last two bytes respectively represent Pseudonode ID and LSP number.
+            A pseudonode is a logical representation of the LAN which is generated by a Designated Intermediate System (DIS)
+            on a LAN segment. If one LSP is not fit by maximum LSP size then thats sent in another LSP with LSP number
+            is incremented by 1.
+            A router's learned LSP gets refreshed by 'remaining_lifetime'. Then the sequence number is incremented by 1.
+          type: array
+          items:
+            type: string
+            format: hex
+  
+    IsisLsps.State:
+      description: >-
+        The result of ISIS LSP information that are retrieved.
+      type: object
+      properties:
+        isis_router_name:
+          description: >-
+            The name of a ISIS Router.
+          type: string
+        lsps:
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.State'
+
+    IsisLsp.State:
+      description: >-
+        ISIS LSP.
+      type: object
+      properties:
+        lsp_id:
+          description: >-
+            LSP ID of the LSP, e.g. '6400000000010000'.
+          type: string
+          format: hex
+        pdu_type:
+          description: >-
+            Link State PDU type.
+          type: string
+          enum: [level_1, level_2]
+        remaining_lifetime:
+          description: >-
+            Remaining lifetime in seconds before the LSP expiration.
+          type: integer
+          format: int64
+          minimum: 0
+          maximum: 4294967295
+        sequence_number:
+          description: >-
+            Sequence number of the LSP.
+          type: integer
+          format: int64
+          minimum: 0
+          maximum: 4294967295
+        pdu_length:
+          description: >-
+            Total length of the LSP.
+          type: integer
+          minimum: 0
+          maximum: 65535
+        flags:
+          description: >-
+            LSP Type-Block flags.
+          $ref: '#/components/schemas/IsisLsp.Flags'
+        is_type:
+          description: |-
+            IS Type - bits 1 and 2 indicate the type of Intermediate System.
+            1 - ( i.e. bit 1 set) Level 1 Intermediate system.
+            2 - Unused value.
+            3 - (i.e. bits 1 and 2 set) Level 2 Intermediate system. 
+          type: integer
+          minimum: 1
+          maximum: 3
+        hostname:
+          description: >-
+            This container defines TLV 137.
```

Comment:
Not a container. Also, the yang model shows a list of host name(s) - so either of the models should be corrected for consistency.

Reply:
ISIS Yang-model referes as array. But our isis router can have  only hostname. will change to non-array there.

Reply:
Can we check the RFC/spec . If it allows multiple we should keep as multiple. Maybe send only first name initially . When fixed and if spec allows, send multiple. 
But should be spec driven and not our implementation,